README.md

```diff
@@ -23,24 +27,25 @@ The repository consists of the following Java classes:
 The repository consists of the following scripts and resources: 
 
 1. src/test/resources data files for the test classes 
-2. dataflow-staging-policy.json - a policy for expiring objects in the bucket used for staging the dataflow jobs
-3. run-simple.example - example script for running the simple pipeline using maven
-4. run-nested.example - example script for running the nested pipeline using maven
-5. pom.xml - maven build script 
+1. dataflow-staging-policy.json - a policy for expiring objects in the bucket used for staging the dataflow jobs
+1. run.sh - example script for running the pipelines using maven
+1. pom.xml - maven build script 
  
 
 ## Getting Started
 
-1. copy the files run-simple.example and run-nested.example to run-simple and run-nested respectively.
-'''
-cp run-simple.example run-simple
-cp run-nested.example run-nested 
+1. Set your environmet variables.  For example: 
+
+     ```shell
+     export PROJECT_ID=[YOUR_PROJECT_ID]
+     export ZONE=[CHOOSE_AN_APPROPRIATE_ZONE] # e.g. us-east1-c
+     export STAGING_BUCKET=${PROJECT_ID}-etl-staging-bucket
+     export DATASET=musicbrainz
+     export DESTINATION_TABLE=recordings_by_artists
+     ```
 
-2. Edit each file replacing the  #STAGING_BUCKET_, #PROJECT, #DATASET with the values specific to your account. 
-3. Edit each file replacing #DESTINATION_TABLE with the table you want to load the denormalized data into.  Note that to preserve the table for each run you may want to use different destination tables for each of these scripts. 
-4. Save the changes for each script.
-5. Run either script as desired.
-6. As an alternative to editing the script you can simply copy and paste the command therein to your shell replacing the aforementioned values with those specific to your project.
+1. Run the script `run.sh` as desired with either the `simple` or `nested` option.
+1. As an alternative to editing the script you can simply copy and paste the command therein to your shell replacing the aforementioned values with those specific to your project.
 
```

Comment:
Same here.

Reply:
Can we add the examples of the run script here formatted?

Reply:
Also let's see if we can get the service account instructions included here.

Reply:
I've modified the README to show the command.  For the service account, I propose we just include the link:  https://cloud.google.com/dataflow/docs/quickstarts/quickstart-java-maven#before-you-begin